pkg/reconcile/pipeline/infinispan/handler/manage/conditions.go

```diff
@@ -41,22 +41,27 @@ func PreliminaryChecks(i *ispnv1.Infinispan, ctx pipeline.Context) {
 	}
 }
 
+var cacheServiceForbiddenMsg = "Please change to service type DataGrid. CacheService is no longer supported."
```

Comment:
If we want this to be a separate var, than we should mark this as a `const`.

Personally I would just declare this inline as it's only used once in the function, e.g.:

```go
i.SetCondition(
  ispnv1.ConditionWellFormed,
  metav1.ConditionFalse,
  "Please change to service type DataGrid. CacheService is no longer supported."
)
```

That's your call though :slightly_smiling_face:

Reply:
ok to remove, it was just a relic from a previous version thanks